skins/cat17/src/scripts/main.js

```diff
@@ -127,14 +127,6 @@
                 if (currentScroll + 70 >= fixBarTop) {
                     $('.state-bar').addClass('active');
                     $('.fixed-button').addClass('active');
-                    $('.menu-main').addClass('under-bar');
-
-                    if ($('.footer').isVisible('top')) {
-                        $('.menu-main').removeClass('under-bar');
-                    } else {
-                        $(".state-bar").addClass('active');
```

Comment:
Can not fathom how this ever helped us. Do you have a theory?

Reply:
My theory is that it was meant to avoid the right state "bar" to overlap the footer. But that is already taken care of in another place and this is just a vestige.